PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/layers/receiver.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include "./globals.h"
+#include "./layersWrapper.h"
+#include "../utils/queue_wrapper.h"
+#include "flit.h/"
+
+class Receiver {
+public:
+	Globals globals;
+	LayersWrapper layers;
+	QueueWrapper<Flit> listenOn;
```

Comment:
This was removed from the class diagram because the thread will be listening, not this class